test/Test/Compile/Golden/bounded/bounded-output.json

```diff
@@ -1,130 +1,149 @@
-{
-  "tag": "Quant",
-  "contents": [
-    "All",
-    "x",
-    [],
```

Comment:
Unfortunately this file is part of the test suite, so changing it will cause tests to fail. Could you create your own local copy of this file until we get around to updating the JSON generation to actually produce these changes?